capture-sdk/sdk/src/main/java/net/gini/android/capture/network/FailureException.java

```diff
@@ -0,0 +1,12 @@
+package net.gini.android.capture.network;
+
+import net.gini.android.capture.error.ErrorType;
+
+public class FailureException extends Exception {
```

Comment:
[Request] We can pass the `ErrorType` around without having to wrap it into an exception. I didn't find any place where we throw a `FailureException` and currently it only serves as a wrapper.

Reply:
Sorry, for not seeing that we need the FailureException due to Future requiring exception types.